tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -270,19 +270,19 @@ def test_count_transaction_with_valid_condition(self):
                 MetricField(
                     op="count_transaction_name",
                     metric_mri=TransactionMRI.DURATION.value,
-                    params={"condition": "is_unparameterized"},
+                    params={"identifier": "is_unparameterized"},
                     alias="count_transaction_name_is_unparameterized",
                 ),
                 MetricField(
                     op="count_transaction_name",
                     metric_mri=TransactionMRI.DURATION.value,
-                    params={"condition": "is_null"},
+                    params={"identifier": "is_null"},
```

Comment:
One final comment, lets pick another name than `identifier` here because I feel it would be confusing as a param name
I would just name this as `transaction_name` so it is explicit